### ISSUE TITLE
Add an associated caption field to Featured Image field on News Content Types

### ISSUE THREAD

**Issue by michaelablackham** [massgov/support-test#292]
**Is your feature request related to a problem? Please describe.**
For accessibility reasons, it is recommended to have a caption available for all images. This allows for image credits, and for "long alt text" to have an appropriate place to live since most instances of ong alt text has great information in it that should be available to everyone. Right now, many people are adding massive long alt text with image credits into the feature image alt text, which is a no-no for a11y and doesn't allow for image credits to be seen.

**Describe the solution you'd like**
I would like an associated caption field with the Featured Image Field (Main content > Featured Image).
Captions are available with rich text images, but not this featured image.
Visually this would look like other captions on the site.

**Additional context**

<img width="1289" height="449" alt="Image" src="https://github.com/user-attachments/assets/3df904b5-8459-4acf-985f-802b5921f328" />
.

**Reference**
Example pages where this would be beneficial:
1. [News Content Type: At Chop Chaque Bogs, Retired Cranberry Farm Sees New Life as Restored Wetlands](https://edit.mass.gov/news/at-chop-chaque-bogs-retired-cranberry-farm-sees-new-life-as-restored-wetlands)
2. [New Content Type: Healey-Driscoll Administration Awards $325K to Restore Critical Wildlife Habitat](https://edit.mass.gov/news/healey-driscoll-administration-awards-325k-to-restore-critical-wildlife-habitat)
